src/test/kotlin/org/jetbrains/intellij/IntelliJPluginSpec.kt

```diff
@@ -71,8 +73,8 @@ class IntelliJPluginSpec : IntelliJPluginSpecBase() {
                 SIGN_PLUGIN_TASK_NAME,
                 VERIFY_PLUGIN_TASK_NAME,
                 VERIFY_PLUGIN_CONFIGURATION_TASK_NAME
-            ),
-            tasks(PLUGIN_GROUP_NAME),
+            ).joinToString("\n"),
+            tasks(PLUGIN_GROUP_NAME).joinToString("\n"),
```

Comment:
this is a minor change, slightly related to this PR, to make a test failure more understandable with the `<Click to see difference>` feature in IntelliJ

<img width="773" alt="image" src="https://user-images.githubusercontent.com/897017/193699873-91852f0a-2356-4998-b317-38d5bc21d8d1.png">